package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@seatgeek/node-hcl",
   "version": "0.0.0-semantically-released",
-  "packageManager": "yarn@4.1.1",
```

Comment:
you're able to drop from yarn 4 to yarn 1 without any major breaking changes?

also - why the drop? if semver uses npm over yarn maybe we should just use npm here, using yarn + semver in the other repo has been a major pain

Reply:
Yup it was able to build/test successfully dropping down, since the package isn't too complicated. Did the drop since the Github Action global yarn package is 1.22.22, so decided to also use that as well.

You make a good point about moving over to npm. Everything about packaging is handled using npm, so we might as well use npm across the board. I'll update.

Reply:
Updated